plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreTable.java

```diff
@@ -56,6 +60,35 @@ public abstract class PostgreTable extends PostgreTableReal
 {
     private static final Log log = Log.getLog(PostgreTable.class);
 
+    /**
+     * see https://www.postgresql.org/docs/8.2/ddl-system-columns.html
+     * 'oid' described at {@link PostgreConstants#PSEUDO_ATTR_OID}
+     */
+    private static final Map<DBDPseudoAttributeType, List<Pair<String, String>>> ALWAYS_PRESENTED_PSEUDO_ATTR_ENTRIES = Map.of(
```

Comment:
In fact these attributes are real. We skip them in columns read method. But we can read them from database. Maybe we should (read + hide)?